pkg/services/sqlstore/migrations/annotation_mig.go

```diff
@@ -79,13 +79,20 @@ func addAnnotationMig(mg *Migrator) {
 		},
 	}
 
-	mg.AddMigration("Create annotation_tag table v2", NewAddTableMigration(annotationTagTable))
-	mg.AddMigration("Add unique index annotation_tag.annotation_id_tag_id", NewAddIndexMigration(annotationTagTable, annotationTagTable.Indices[0]))
+	annotationTagTableDoesNotExist := &IfNotTableExistsWithoutPrimaryKey{TableName: annotationTagTable.Name, ColumnName: "id"}
```

Comment:
rename `annotationTagTableDoesNotExist` -> `annotationTagTableDoesNotContainPrimaryKey`

Reply:
renamed appropriately in 03e12ac7583fd2f78efcbf82d953c7fd46431d2e